eng/targets/Helix.Common.props

```diff
@@ -18,23 +18,22 @@
     <HelixAvailableTargetQueue Include="Windows.81.Amd64.Open" Platform="Windows" />
     <HelixAvailableTargetQueue Include="Windows.7.Amd64.Open" Platform="Windows" />
     <HelixAvailableTargetQueue Include="Windows.10.Amd64.EnterpriseRS3.ASPNET.Open" Platform="Windows" EnableByDefault="false" />
-    <HelixAvailableTargetQueue Include="OSX.1013.Amd64.Open" Platform="OSX" />
-    <HelixAvailableTargetQueue Include="Ubuntu.1604.Amd64.Open" Platform="Linux" />
+    <HelixAvailableTargetQueue Include="OSX.1014.Amd64.Open" Platform="OSX" />
+    <HelixAvailableTargetQueue Include="OSX.1100.Amd64.Open" Platform="OSX" />
```

Comment:
Is it fine / worthwhile to add the OSX.1100.Amd64.Open queue❔

Reply:
Doesn't seem worth adding more coverage in an old release especially since we don't generally see many issues due to OSX differences (basically we get more noise for no reward).  But if you are upgrading, maybe just pick one? 1100?

Reply:
Removed